debian/compat

```diff
@@ -1 +1 @@
-9
```

Comment:
why this change ?

Reply:
https://github.com/citusdata/packaging/runs/3720189231?check_suite_focus=true#step:6:403
Gave warning in packaging process
So changed in all of them